daemon/src/model/cfd.rs

```diff
@@ -522,26 +516,31 @@ impl Cfd {
             return Err(CannotRollover::WasJustRolledOver);
         }
 
-        // only state open with no attestation is acceptable for rollover
-        // TODO: Rewrite it terms of events
-        // if !matches!(
-        //     self.state(),
-        //     CfdState::Open {
-        //         attestation: None,
-        //         ..
-        //     }
-        // ) {
-        //     // TODO: how to derive state for these messages?
-        //     return Err(CannotRollover::WrongState {
-        //         state: "Insert state here (how do to it?)".into(),
-        //     });
-        // }
-
         Ok(())
     }
 
-    fn can_roll_over(&self) -> bool {
-        self.lock_finality && !self.commit_finality && !self.is_final() && !self.is_attested()
+    pub fn is_rollover_possible_maker(&self) -> Result<(), CannotRollover> {
+        self.can_roll_over()
+    }
+
+    fn can_roll_over(&self) -> Result<(), CannotRollover> {
```

Comment:
True. I was unsure, because for e.g. collab settlement we use `can`. Happy to change it to `is` for concistency here. That might need another round of cleanup at some point.